internal/pkg/config/profiles.go

```diff
@@ -39,6 +41,98 @@ func GetProfile() (string, error) {
 	return profile, nil
 }
 
+// GetProfileFromEnv returns the profile from the environment variable.
+// If the environment variable is not set, it returns an empty string.
+// If the profile is not valid, it returns an error.
+func GetProfileFromEnv() (string, bool) {
+	return os.LookupEnv(ProfileEnvVar)
+}
+
+// CreateProfile creates a new profile.
+// It creates a new folder for the profile and copies the config file from the current profile to the new profile.
+// If the fromDefault flag is set, it creates an empty profile.
+// If the setProfile flag is set, it sets the new profile as the active profile.
+// If the profile already exists, it returns an error.
```

Comment:
```suggestion
// CreateProfile creates a new profile.
// If emptyProfile is true, it creates an empty profile. Otherwise, copies the config from the current profile to the new profile.
// If setProfile is true, it sets the new profile as the active profile.
// If the profile already exists, it returns an error.
```